Computational biologist.
Extensive experience in the processing and statistical analysis of next generation sequencing data, for single cell, bulk and spatial genomics libraries.
Member of ENCODE and Roadmap Epigenomics.
Experience with: [sc]ATAC-, DNase-, [sc]RNA-, Small RNA-, 3/4/5/HiC-, TF-ChIP-, HistMod-ChIP-, MeDIP-, and DNAmethyl-seq.
Solid publication record on functional genomics and epigenomics, focused on the integration of disparate datasets to understand and unravel new genome-wide principles of transcriptional and post transcriptional regulation in mammalian systems.
Have been using machine and statistical learning since my Ph.D. in proteomics.
~20 years programming experience: R (with Bioconductor), Python, Matlab, Perl, SQL, Java, C/C++, Lisp, HTML, PHP.
Trained in Spark/big data analytics platforms for use in biological data storage and analysis.
Using deep learning models for regulatory genomics and single cell data representations.
Married to Erin M. González, father of Victoria Leena and Antonio David.
Live in Livermore, CA, and love Sauced BBQ and Spirits.
We are Christians and members of the Valley Christian Center community.
Love to play fútbol and follow La Liga.
Grew up in Cali, Colombia, and attended college in Bogotá, Colombia.
US citizen.

# Employment and Education

## 2018-2020 | Senior Computational Biologist | 10x Genomics
Wrote software to help unravel biology and human health. Developer for the single cell ATAC product (<code>cellranger-atac</code>) and the Visium/Spatial Transcriptomics product (<code>spaceranger</code>), implemented in Python and Rust. 

## 2015-2018 | Bioinformatician | Epinomics
Implemented and trained ML models to decode the programming of our genome -- the epigenome -- to drive personalized medicine, impact drug development and discover leads into clinical applications. Company was acquired by 10x Genomics.

## 2012-2015 | Postdoc in the group of Chirstina S. Leslie | Computational Biology Program | MSKCC

- Enhancer dynamics: Integrative analysis of enhancer landscape and gene expression dynamics in hematopoietic differentiation using DNase-seq, histone mark ChIP-seq, and RNA-seq. Using a model that learns chromatin to predict gene expression, we showed how early establishment of enhancers and regulatory locus complexity together govern gene expression changes at cell state transitions.
- Immunology: Using inducible genetic tracing, we analyzed the long-term stability of activation-induced transcriptional, epigenomic, and functional changes in regulatory T cells (Tregs). We found that the inflammation-experienced Treg cell population reversed many activation-induced changes and lost its enhanced suppressive function over time. ‘‘Memory-less’’ Tregs may help avoid a state of generalized immunosuppression. In collaboration with Alexander Rudensky (MSKCC).
- Small RNAs: We show that in vivo, many expressed microRNAs exist in an inactive reserve, not stably associated with target mRNA. External signaling can promote the recruitment of these inactive microRNAs to large mRNA-containing complexes, in which their ability to repress targets is increased. In collaboration with Tom Tuschl (Rockefeller), Craig Thompson, and Andrea Ventura (MSKCC).
- Small RNAs: Study of polycistronic microRNA clusters. A large scale RNA-seq analysis of wild type and mutant embryos that were genetically engineered to harbor targeted deletions of individual components of the miR-17~92 cluster. We show how functional cooperation between members of the microRNA cluster affects gene expression programs in vivo. In collaboration with Andrea Ventura (MSKCC).
- Pharmacogenomics: Using drug response data for cancer cell lines, e.g. CCLE, we developed a multi-task learning strategy to predict drug sensitivity by jointly learning across many drugs. On held-out data, our joint training not only outperforms models that predict response individually for each drug, but also produces more interpretable results that lead to potential drug repurposing targets.
